corrigir (#6)
VitorBSP/Tempo_Tela: relatorio.Rmd

  mypdf1::pdf1_tbl('Proporção de utilização de aplicativos para controlar tempo')
```

> Na sequência é questionado aos estudantes quais são os dispositivos que eles mais utilizam, as respostas estão disponível na Tabela \@ref(tab:tab3). No geral a opção computador/Notebook destacou-se com 53% das respostas. A maioria das mulheres responderam utilizar mais o celular, enquanto que os homens optam pelo computador. Outras opção de resposta para essa questão, como, tablet, televisão e video game não obtiveram nenhuma resposta.

Reviewer: ```suggestion
Na sequência é questionado aos estudantes quais são os dispositivos que eles mais utilizam, as respostas estão disponível na Tabela \@ref(tab:tab3). No geral a opção computador/Notebook destacou-se com 53% das respostas. A maioria das mulheres responderam que utilizam mais o celular, enquanto que os homens optam pelo computador. Outras opção de resposta para essa questão, como, tablet, televisão e video game não obtiveram nenhuma resposta.
```

  mypdf1::pdf1_tbl_freq2(`Sexo:`, `Qual plataforma de Video Games você tende a utilizar mais ?`)
```

```{r bar2, fig.cap="Gráfico de barras em relação aos dias da semana que mais se utiliza o dispositivo"}
df |> 
  separate_rows(`Referente a questão anterior qual(is) é o dia que você mais utiliza o dispositivo?`, sep = ', ') |>   bar_plot(`Referente a questão anterior qual(is) é o dia que você mais utiliza o dispositivo?`)+ 
  theme(legend.position = 'none')
```

> Referente a Figura \@ref(fig:bar2), pode-se concluir que não há uma diferença sugestiva entre os dias uteis da semana (segunda, terça, quarta, quinta, sexta) que mais se utiliza o dispositivo, todos os dias estão com cerca de 16%. Entretanto, é possível notar essa diferença em relação ao fim de semana, sábado e domingo, com cerca de 11% e 10%, respectivamente. Ambos com uma percentagem relativamente menor quando comparado aos dias uteis.

Reviewer: ```suggestion
Referente a Figura \@ref(fig:bar2), pode-se concluir que não há uma diferença expressiva entre os dias úteis da semana (segunda, terça, quarta, quinta, sexta) que mais se utiliza o dispositivo, todos os dias estão com cerca de 16%. Entretanto, é possível notar essa diferença em relação ao fim de semana, sábado e domingo, com cerca de 11% e 10%, respectivamente. Ambos com uma percentagem relativamente menor quando comparado aos dias úteis.
```

```{r bar4, fig.cap="Gráfico de barras referente aos dias da semana e a utilização de plataformas de streaming"}
df |> 
  separate_rows(`Qual(is) dia(s) da semana você tende a passar mais tempo em plataformas de streaming :`, sep = ', ') |>   bar_plot(`Qual(is) dia(s) da semana você tende a passar mais tempo em plataformas de streaming :`)+ 
  theme(legend.position = 'none')
```

Em relação a Figura \@ref(fig:bar4), observa-se novamente que os estudantes tendem a utilizar plataformas de streaming durante o fim de semana, domingo com cerca de 24% das respostas e sábado com 23%, seguidos por sexta-feira com 15%.

```{r bar5, fig.cap="Gráfico de barrras relativo aos dias da semana e a utilização de plataformas de estudo"}
df |> 
  separate_rows(`Qual(is) dia(s) da semana você tende a passar mais tempo em plataformas de estudo :`, sep = ', ') |>   bar_plot(`Qual(is) dia(s) da semana você tende a passar mais tempo em plataformas de estudo :`)+ 
  theme(legend.position = 'none')
```

> Em contrapartida, percebe-se que na Figura \@ref(fig:bar5), os estudantes tendem a passar mais tempo em plataformas de estudo, em dias uteis da semana, quarta-feira com 19% das respostas, seguido pelos outros dias, o final de semana fica com as menores percentagens, ou seja as plataformas de estudo são menos utilizadas durante esses dias.

Reviewer: ```suggestion
Em contrapartida, percebe-se que na Figura \@ref(fig:bar5), os estudantes tendem a passar mais tempo em plataformas de estudo, em dias úteis da semana, quarta-feira com 19% das respostas, seguido pelos outros dias, o final de semana fica com as menores percentagens, ou seja as plataformas de estudo são menos utilizadas durante esses dias.
```

De acordo com a Figura \@ref(fig:bar9), ao serem questionado sobre a relação entre o aumento do tempo de tela com o aprendizado de novas ferramentas, cerca de 36% dos estudantes responderam que houve melhora.

```{r bar12, fig.cap="Gráfico de barras em relação a sensação sobre a influencia do aumento de tempo de tela na cooperação em trabalhos em equipe"}
df |> filter(!`De modo geral você acredita que seu tempo de tela aumentou durante a pandemia?...28` == "Não") |>
  bar_plot(`Agora em relação ao tempo de tela, o quanto você acha que a pandemia influenciou  na sua cooperação nos trabalhos em equipes:`)+
  theme(legend.position = 'none')
```

> Na Figura \@ref(fig:bar12), nota-se que não há diferenças expressivas entre as alternativas: **Pior, Indiferente, e Melhor**, com 27%, 29% e 23% das respostas, respectivamente, esses porcentagens dizem respeito quando questionado  a influencia do tempo de tela em relação a cooperação dos estudantes no trabalhos em equipe. 

Reviewer: ```suggestion
Na Figura \@ref(fig:bar12), nota-se que não há diferenças consideráveis entre as alternativas: **Pior**, **Indiferente**, e **Melhor**, com 27%, 29% e 23% das respostas, respectivamente, essas porcentagens dizem respeito quando questionado  a influencia do tempo de tela em relação a cooperação dos estudantes no trabalhos em equipe. 
```

### Testes preliminares

Começaremos com o teste de Bartlett para verificar correlação, sendo as hipóteses testadas são de que
$$H_0:  \mbox{ A matriz de correlação é igual a matriz de identidade} $$ versus a hipótese alternativa $$H_1: \mbox{A matriz de correlação não é igual a matriz de identidade}$$

```{r}
cor = df1 |> rename(Aprendizado = `Aprendizado de novas ferramentas`, Metas = `Cumprimento de Metas`) %>% .[,21:26] %>%
  cor()
p1 <- psych::cortest.bartlett(cor, n = 110)$p.value
```

> Pelo teste de Bartlett, com índice de significância de 5% tem-se que o p-valor é `r p1`, ou seja, menor que $\alpha$ (0,05), então, há evidências para rejeitar a hipótese nula, ou seja, há evidências para rejeitar que a matriz de correlação seja igual a matriz identidade.

Reviewer: ```suggestion
Pelo teste de Bartlett, com índice de significância de 5% tem-se que o p-valor é `r p1`, ou seja, menor que $\alpha$ (0,05), então há evidências para rejeitar que a matriz de correlação seja igual a matriz identidade.
```

Do mesmo modo podemos utilizar o Alfa de Cronbach para estimar a confiabilidade e a consistência interna de um teste, questionário ou escala de uma pesquisa, pois mede a  correlação entre respostas em um questionário, assim obtivemos um Alfa de Cronbach de `r cronbach` o que o caracteriza como aceitável de acordo com George e Mallery (2003). Assim na Figura \@ref(fig:gcor), podemos analisar como as questões se relacionam.

```{r gcor, fig.cap="Correlograma das questões com escala Likert"}
ggcorrplot::ggcorrplot(cor, hc.order = TRUE, 
           type = "lower", 
           lab = TRUE, 
           lab_size = 3, 
           method="circle", 
           colors = c("tomato2", "white", "springgreen3"), 
           title="Correlograma das questões com escala Likert", 
           ggtheme=theme_bw)

```

O correlograma da Figura \@ref(fig:gcor), indica que rendimento e concentração foram as opções mais correlacionadas, provavelmente por apresentarem semelhança nas respostas, o que pode indicar que os estudantes que consideraram piora na sua concentrção, também identificaram no rendimento, assim como o contrário (melhora).

### Análise Fatorial com Componentes Principais

> Seguiremos a análise empregando o método dos componentes principais da análise fatorial com a matriz de correlação amostral com a amostra de 110 consumidores avaliando as 6 perguntas com notas de 1 a 5, lidando com a escala likert, no sentido de reduzir o número das 6 variáveis.

Reviewer: 110 consumidores né

> Seguiremos a análise empregando o método dos componentes principais da análise fatorial com a matriz de correlação amostral com a amostra de 110 consumidores avaliando as 6 perguntas com notas de 1 a 5, lidando com a escala likert, no sentido de reduzir o número das 6 variáveis.

Author: ```suggestion
Seguiremos a análise empregando o método dos componentes principais da análise fatorial com a matriz de correlação amostral com a amostra de 110 estudantes avaliando as 6 perguntas com notas de 1 a 5, lidando com a escala likert, no sentido de reduzir o número das 6 variáveis.
```
